include/NovelRT.Interop/Audio/NrtAudio.h

```diff
@@ -0,0 +1,9 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_INTEROP_AUDIO_H
+#define NOVELRT_INTEROP_AUDIO_H
+
+#include "NrtAudioService.h"
+
+#endif // NOVELRT_INTEROP_AUDIO_H
```

Comment:
Empty header? Forget to add headers here?

Reply:
This is not strictly empty, but it has been moved to match how `NrtEcs.h` is formatted and utilised. Please let me know if you'd expect something different here? Even though there's only one header being included, I thought we may rather keep consistency here instead of changing it depending on the namespace.

Reply:
Oh, I misread it in that case. I don't remember seing any includes previously.